python/mujinwebstackclient/controllerwebclientraw.py

```diff
@@ -157,7 +157,20 @@ class ControllerWebClientRaw(object):
     _subscriptionLock: threading.Lock  # Lock protecting _webSocket and _subscriptions
     _backgroundThread: BackgroundThread = None  # The background thread to handle async operations
 
-    def __init__(self, baseurl: str, username: str, password: str, locale: Optional[str] = None, author: Optional[str] = None, userAgent: Optional[str] = None, additionalHeaders: Optional[Dict[str, str]] = None, unixEndpoint: Optional[str] = None) -> None:
+    _threadName: Optional[str] = None
```

Comment:
Please add a comment

Reply:
Added in https://github.com/mujin/mujinwebstackclientpy/pull/75/commits/0b276ec5548277f63f4217b3a4f7f2f586d65cda